src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -160,10 +160,10 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
   const module = {} as typeof Module;
   const suppressMessages = ['DEBUGGING ENABLED'];
 
-  module.print = line => (suppressMessages.indexOf(line) < 0 && console.log(`WASM: ${line}`));
+  module.print = line => (suppressMessages.indexOf(line) < 0 && console.log(line));
 
   module.printErr = line => {
-    console.error(`WASM: ${line}`);
+    console.error(line);
```

Comment:
I removed the `WASM: ` prefix because it serves no purpose. Developers who call `Console.Log` should be able to choose exactly what string gets logged, without the framework enforcing an arbitrary prefix.